frontend/public/components/sidebars/resource-sidebar.jsx

```diff
@@ -85,22 +89,52 @@ const ResourceSamples = ({ samples, kindObj, downloadSampleYaml, loadSampleYaml
   />
 );
 
+const ResourceSnippets = ({ snippets, kindObj, insertSnippetYaml }) => (
```

Comment:
pls move to separate component file

Reply:
Same issue here with `public` folder and lots of components.

Reply:
I totally, agree. Then we can keep the components as-is and later think of a refactoring around the admin code.